File: leetcode-submissions/650_2_Keys_Keyboard.cpp
class Solution {
public:
    int minSteps(int n) {
        int prev=0;
        int curr=1;
        int ans=0;
        while(curr<n){
            if(n%curr==0){
                prev=curr; //copy 
                curr+=prev;//paste
                ans+=2;
            }else{
                ans++;
//paste
curr+=prev;
            }
        }
        return ans;
    }
};

File: leetcode-submissions/650_2_Keys_Keyboard.ts
function minSteps(n: number): number {
    let curr:number =1;
    let prev:number =0;
    let ans:number =0;
    while ( curr<n){
if(n%curr===0){
prev=curr;
curr+=prev;
ans+=2;
}else{
    curr+=prev;
    ans++;
}
    }
return ans ;
};